.github/workflows/tests_primary.yml

```diff
@@ -120,7 +120,8 @@ jobs:
       with:
         node-version: ${{matrix.node-version}}
         command: npm run ttest -- --shard ${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
-        bot-name: "${{ matrix.os }}-node${{ matrix.node-version }}-${{ matrix.shardIndex }}"
+        bot-name: "${{ matrix.os }}-node${{ matrix.node-version }}"
```

Comment:
wouldn't it break the merge via adding all the shard zips into the archive with the same name?